### PR TITLE
Create/weekly routine page

### LINKED ISSUE #4: Create weekly routine page
## Value Statement

As a user, I want to view my weekly routine, so that I have an overview

## Description

Screenshot 2021-04-26 at 12 06 54

## Acceptance Criteria

* The "+" button takes the user to the product list page
* If products are already added, the page displays cards for every weekday with the routines for that day's morning and evening

## Tasks

- [x] Create feature branch
- [x] Create week day component
- [x] Get data from local storage to check if anything has been added and render data
- [x] Add navigation route for "+" button
- [x] Add styling

## Complexity
medium

### PR DESCRIPTION
**Pull request by FriederikeM**
* Statically create weekly routine page and weekday card component
* Style page and component

### CODE REVIEW

File: src/components/WeekDayCard.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          <section className="evening">
+            <ul className="product-name-list">
+              <li className="product-name-right">Squalane Cleanser</li>
+              <li className="product-name-right">Argireline Solution 10%</li>
+              <li className="product-name-right">Hyaluronic Acid 2% + B5</li>
+              <li className="product-name-right">
+                Natural Moisturizing Factors + HA
+              </li>
+              <li className="product-name-right">
+                100% Organic Cold-Pressed Rose Hip Seed Oil
```
Comment (soerendolling) [inline src/components/WeekDayCard.js:37]: It so screams like, its good for you

**Review by soerendolling — APPROVED** [review]
Looks all fine to me

File: src/components/WeekDayCard.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.evening {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
```
Comment (yvlange) [inline src/components/WeekDayCard.css:94]: You made it work to appear on the right side!!!!! Amazing job!! It just looks fantastic 🎉

File: src/color.css [hunk 3]
```diff
@@ -7,11 +7,15 @@
   --color-three-opaque: #a267699f;
   --color-four: #e0c2c0;
   --color-four-opaque: #e0c2c07c;
+  --color-four-opaque-dark: #a8868391;
```
Comment (yvlange) [inline src/color.css:10]: Maybe just maybe make the colour a bit darker so the icons pop out a bit more 🤔

File: src/components/WeekDayCard.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import "./WeekDayCard.css";
+import { HiSun } from "react-icons/hi";
+import { FaMoon } from "react-icons/fa";
+
+export default function WeekDayCard() {
+  return (
+    <div className="WeekDayCard">
+      <div className="weekday">
+        <h4 className="weekday-headline">monday</h4>
```
Comment (yvlange) [inline src/components/WeekDayCard.js:9]: the font-colour is black on my screen right now. How does it look if you do the same colour as on the header?

**Review by yvlange — APPROVED** [review]
Love the design 😍 just had some small ideas. Btw the animations from your product list looks amazing!!

**Pull request merged**